upf/upf_cli.c

```diff
@@ -1503,6 +1503,9 @@ upf_policy_command_fn (vlib_main_t * vm,
   u8 *policy_id;
   u8 action = 0;
 
+  if (!unformat_user (main_input, unformat_line_input, line_input))
+    return 0;
+
```

Comment:
Why invoke `unformat_user()` twice?
Other place the input is handled likewise, for reference: https://github.com/travelping/upg-vpp/blob/master/upf/upf_cli.c#L60-L63

Reply:
good point, fixed